package.json

```diff
@@ -1312,6 +1312,11 @@
           "type": "boolean",
           "scope": "resource"
         },
+        "objectscript.serverSourceControl.respectEditableStatus": {
```

Comment:
Can we add an explicit default of `false`?

Reply:
Done, and also for other boolean properties where it was previously implicit.